proto/table.proto

```diff
@@ -307,3 +313,32 @@ message TableImportFromTableCommand {
    */
   ImportExportShared.ImportOptions importOptions = 3;
 }
+
+/**
+ * Command is used when export file from Keboola connection
+ * Command will take backend table and export it into destination file based on provided options
+ */
+message TableExportToFileCommand {
```

Comment:
Ted jeste koukam na [dokumentaci API](https://keboola.docs.apiary.io/#reference/tables/unload-data-asynchronously/asynchronous-export) - mely by tu byt i dalsi parametry?
- limit
- changed*
- columns
- where*
- orderBy

Nebo jsou az do dalsi faze?

Reply:
řešil bych to stejně jako u preview, phase 2 založ na to pls task

Reply:
[rozdeleno](https://keboola.atlassian.net/browse/KBC-2615)